conference.js

```diff
@@ -874,6 +876,39 @@ export default {
         APP.store.dispatch(handleToggleVideoMuted(mute, showUI, ensureTrack));
     },
 
+    /**
+     * Simulates blurred background selection/removal on video background. Used by API only.
+     * @param {boolean} [blurred]
+     */
+    setBlurredBackground(blurred) {
```

Comment:
TR his code needs to be in features
https://github.com/jitsi/jitsi-meet/tree/master/react/features/virtual-background

Reply:
I don't need it at the moment but it could be an interesting feature. I'll add it when moving the code into virtual-background

Reply:
Code moved